Makefile

```diff
@@ -0,0 +1,16 @@
+CPPFLAGS:=-I./lua/src
+CFLAGS:=--std=c99 -g -Wall -O2 -fPIC -shared
+
+.PHONY: all lua clean
+
+all: lua runtime/tcore.o
+
+lua:
+	cd lua/src; make linux-readline
```

Comment:
Why not use the lua root makefile (instead of the one at src dir)?

Reply:
The linux-makefile target won't work in macos. 😢 
We could have a TARGET variable (and an error if it isn't defined).